src/internal.h

```diff
@@ -27,7 +28,8 @@ bool internalStaticCheck(const SBuf &urlPath);
 char *internalLocalUri(const char *dir, const SBuf &name);
 char *internalRemoteUri(bool, const char *, unsigned short, const char *, const SBuf &);
 const char *internalHostname(void);
-int internalHostnameIs(const char *);
+/// whether the given URI matches this Squid hostname and listening IP:port
+bool internalWithMyHostname(const AnyP::Uri &, const AnyP::PortCfg &);
```

Comment:
Edited your change suggestion to remove a typo `((`.

Reply:
Retaining the `internalFoo()` naming scheme since this is part of the `internal.h` API. If/When it is moved elsewhere it can be renamed appropriately.